modules/social_features/social_event/src/Entity/Node/Event.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Drupal\social_event\Entity\Node;
+
+use Drupal\social_event\EventEnrollmentInterface;
+use Drupal\social_node\Entity\Node as Node;
+
+/**
+ * Defines bundle class for event node type.
+ */
+final class Event extends Node implements EventInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function isEnrollmentEnabled(): bool {
+    // Get global event settings.
+    $settings = \Drupal::config('social_event.settings');
+    if (
+      $settings->get('disable_event_enroll') ||
+      !$this->hasField('field_event_enroll') ||
+      (!$this->get('field_event_enroll')->isEmpty() && !$this->get('field_event_enroll')->getString())
+    ) {
+      return FALSE;
+    }
+
+    // When field was added to the event the value become `null` which
```

Comment:
Please update the description to say which "field" was added, also please improve the ending part. `if after saving the event the value is 'false'.`

Reply:
added